### PR TITLE
Style Changes

### PR DESCRIPTION
**Pull request by MinchinWeb**
This is all the style changes split out of PR #52.

There should be no substantial code changes, just style fixes.

Issues addressed:

- PEP8 (E125, E127, E128, E201, E202, E203, E221, E225, E231, W291, W293, E301, E302, E303, W391, E501 (selected lines only), E502, E703, and E711)
- Travis-CI
    - add Python 3.5
    - add Python 3.2, PyPy, and PyPy3 as an 'acceptable failure', allowing interested parties to see how close `topydo` is to running on those platforms
- tests
    - rename numbered tests so they sort in order
    - make all of the test file self-runnable
    - rename test files so they start with 'test'. This allows `unittest`'s default auto-discovery to work
- documentation
    - update the copyright year in the version statement
   - update the copyright year in the header of the code files
   - spellcheck
- imports
    - sort imports
    - remove unused imports

### CODE REVIEW

**Comment by bram85** [comment]
Thanks for your patch. Had a quick look at it, seems to look good overall. I'll be out of town for a couple of days, will review it more thoroughly somewhere next week.

**Comment by MinchinWeb** [comment]
Sounds great :+1:

File: test/TestAddCommand.py [hunk 1]
```diff
@@ -41,219 +41,276 @@ def setUp(self):
...
-        self.assertEqual(self.todolist.todo(1).source(), self.today + " Foo p:1")
-        self.assertEqual(self.todolist.todo(2).source(), self.today + " Bar id:1")
+        self.assertEqual(self.todolist.todo(1).source(),
+                         self.today + " Foo p:1")
+        self.assertEqual(self.todolist.todo(2).source(),
+                         self.today + " Bar id:1")
         self.assertEqual(self.errors, "")
 
-    def test_add_dep4(self):
+    def test_add_de04(self):
```
Comment (bram85) [inline test/TestAddCommand.py]: That's a typo here.

**Comment by bram85** [comment]
Looks great, will merge it despite the minor typo. Thanks a lot for your effort to fix those style issues.

**Pull request merged**